homework-submission/myapp/src/stores/rashadStore.js

```diff
@@ -0,0 +1,93 @@
+import { configure, action, observable, runInAction, computed } from "mobx";
+import Todos from "../components/todos.json";
+
+configure({ enforceActions: true });
+
+const url = "https://hyf-react-api.herokuapp.com";
+
+class RashadStore {
+  @observable todosList = [];
+
+  fetchAll = () => {
+    fetch(`${url}/todos`)
+      .then(response => response.json())
+      .then(todosList => {
+        console.log(todosList);
+        runInAction(() => {
+          this.todosList = Todos;
+        });
+      })
+      .catch(err => console.log(err));
+  };
+
+  @action
+  checker = item => {
+    item.done = !item.done;
+    fetch(`${url}/todos/${item._id}`, {
+      method: "PATCH",
+      body: JSON.stringify({ done: item.done }),
+      headers: { "content-type": "application/json" }
+    })
+      .then(response => response.json())
+      .then(todo => {
+        const newTodos = this.todosList.map(element => {
```

Comment:
Why are you looping on your `todos.json` file content while you are connected to the server?